r2dbc-spi-test/src/main/java/io/r2dbc/spi/test/MockColumnMetadata.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017-2019 the original author or authors.
```

Comment:
Don't think this one should have been changed.

Reply:
Let me revert that change.